src/from_numpy.rs

```diff
@@ -1,7 +1,7 @@
 use nalgebra::base::{SliceStorage, SliceStorageMut};
 use nalgebra::{Dynamic, Matrix};
 use numpy::npyffi;
-use numpy::npyffi::objects::PyArrayObject;
+use numpy::{npyffi::objects::PyArrayObject, Element};
```

Comment:
It looks like you're using `numpy::Element` explicitly throughout the file. I like the explicitness of that, but I think that makes this an unused import.

```suggestion
use numpy::{npyffi::objects::PyArrayObject};
```